web/templates/navbar.hbs

```diff
@@ -3,9 +3,9 @@
     <div class="top_navbar_full_width">
         <nav class="header-main" id="top_navbar">
             <div class="column-left">
-                <a class="brand no-style" href="#">
+                <span class="brand no-style">
                     <img id="realm-logo" src="" alt="" class="nav-logo no-drag"/>
-                </a>
+                </span>
```

Comment:
Hmm, I think it's nice that hovering this has the browser show the URL it's going to navigate to is the main page; could we leave it as an `a` with an `href=""` or something?

Reply:
`href=""` will reload the page.
not having an `href` attribute doesn't seem to work to show a URL.

Reply:
Oh! `href=""` seems to work fine. I am not sure why it didn't work earlier, I am sure I tried it.

Reply:
The click handler with `preventDefault` that you added is important.

Reply:
Haha, correct, that's why.